### ISSUE TITLE
Add command-line functionality

### ISSUE THREAD

**Issue by donnell794** [MaxMcGlinnPoole/TermDocumentTensor#9]
Any ideas on what command-line options and flags we should implement?

**Comment by MaxMcGlinnPoole** [comment]
some ideas:

-d "name of directory to read files from"
-t "read text data"
-b "read binary data"
-v "visualize the data, should take in name of the visualization"
-parafac "use PARAFAC decomposition"
-tucker "use Tucker3 decomposition"
-o "produce csv output, this will probably need to be more complicated"

I would recommend looking into argparse for doing this: https://docs.python.org/3/howto/argparse.html

**Comment by MaxMcGlinnPoole** [comment]
Sumanth, what is the update on this?

**Comment by sumanthneerumalla** [comment]
I've added the options, the variable names for these options can be found in the main function where I did the arg parsing.

Sample usage now:  python3 vx.py -d myDirectory -v heatmap -b -parafac -o

Where we specify to parse files in "myDirectory" , output a heatmap, while parsing the files as binary files and using a parafac decomposition on the generated tensor. The -o just stores whether or not to generate an output file.

These variables can start being used in the rest of the code now. I'm in the process of figuring out where to start using them in place of other variables that are used right now.
